src/shogun/mathematics/linalg/LinalgBackendBase.h

```diff
@@ -404,6 +404,55 @@ namespace shogun
 		    BACKEND_GENERIC_IN_PLACE_BLOCK_ELEMENT_PROD, SGMatrix)
 #undef BACKEND_GENERIC_IN_PLACE_BLOCK_ELEMENT_PROD
 
+/**
+ * Wrapper method of in-place vector elementwise division.
+ *
+ * @see linalg::element_div
+ */
+#define BACKEND_GENERIC_IN_PLACE_VECTOR_ELEMENT_DIV(Type, Container)          \
+	virtual void element_div(                                                 \
+	    const Container<Type>& a, const Container<Type>& b,                    \
+	    Container<Type>& result) const                                         \
+	{                                                                          \
+		not_implemented(SOURCE_LOCATION);;                                                    \
+	}
+		DEFINE_FOR_ALL_PTYPE(
+		    BACKEND_GENERIC_IN_PLACE_VECTOR_ELEMENT_DIV, SGVector)
+#undef BACKEND_GENERIC_IN_PLACE_VECTOR_ELEMENT_DIV
+
+/**
+ * Wrapper method of in-place matrix elementwise division.
+ *
+ * @see linalg::element_div
+ */
+#define BACKEND_GENERIC_IN_PLACE_MATRIX_ELEMENT_DIV(Type, Container)          \
+	virtual void element_div(                                                 \
+	    const Container<Type>& a, const Container<Type>& b,                    \
+	    Container<Type>& result) const     \
+	{                                                                          \
+		not_implemented(SOURCE_LOCATION);;                                                    \
+	}
+		DEFINE_FOR_ALL_PTYPE(
+		    BACKEND_GENERIC_IN_PLACE_MATRIX_ELEMENT_DIV, SGMatrix)
+#undef BACKEND_GENERIC_IN_PLACE_MATRIX_ELEMENT_DIV
+
+/**
+ * Wrapper method of in-place matrix block elementwise division.
+ *
+ * @see linalg::element_div
+ */
+#define BACKEND_GENERIC_IN_PLACE_BLOCK_ELEMENT_DIV(Type, Container)           \
+	virtual void element_div(                                                 \
+	    const linalg::Block<Container<Type>>& a,                               \
+	    const linalg::Block<Container<Type>>& b, Container<Type>& result) const                              \
```

Comment:
Do we need/use these?

Reply:
I wanted to remain consistent with `element_prod`.. what do you think @karlnapf